packages/application-components/src/components/tabular-main-page/tabular-main-page.tsx

```diff
@@ -51,40 +52,47 @@ const defaultProps: Pick<TTabularMainPageProps, 'hideControls'> = {
   hideControls: false,
 };
 
-const TabularMainPage = (props: TTabularMainPageProps) => (
-  <PageWrapper>
-    <TabularPageContainer color="surface">
-      <Spacings.Stack>
-        {props.customTitleRow || (
-          <PageHeaderTitle
-            title={props.title}
-            subtitle={props.subtitle}
-            titleSize="big"
+const TabularMainPage = (props: TTabularMainPageProps) => {
+  warning(
```

Comment:
nit: we could also add tests for this case - I attempted doing that [at some point](https://github.com/commercetools/merchant-center-application-kit/blob/bf93463ebd4cb8cf80294dcaaedb115e7adeb8f5/packages/application-components/src/components/tabular-detail-page/tabular-detail-page.spec.tsx#L112:L121)

Reply:
Sure.
I added a new test to both components.

Thanks for the suggestion 👍